### DISCUSSION TITLE
BUG: Randomly switches between dark and light mode

### DISCUSSION THREAD

**Discussion by bateller** [community/community#23995] [Accessibility]
<p>Randomly switching to dark mode even though I have specifically chosen “Light” or “Default to system” (which is always set to “Light”)</p>
<p>I tried clearing my cache, restarting the browser, restarting my system, manually switching between “Dark”, “Light” and “Default to system” while clicking “Update preference” after every change. No luck.</p>
<p>It’s been doing this for 2 days now.</p>
<p>Note in the gif/video it looks like the issue is always resolved with a ‘reload’ (seemingly pointing to a caching issue). However this is not always true. Sometimes I ‘reload’ and it’s still dark, and sometimes I switch to a new page and it’s not dark. Sometimes I can go for 10 minutes without it ever going dark. I haven’t identified a reliable repeating pattern yet.</p>
<p>I’m on macOS Big Sur 11.2 running Chrome Version 88.0.4324.150 (Official Build) (x86_64)</p>
<p><img src="https://user-images.githubusercontent.com/5016178/181080553-ae3895d4-3bcf-4de0-b764-78c620109388.gif" alt="Github_Dark_Light_Switching" data-base62-sha1="h5PDLWA6zn5Ss1XpvaI8JiOhc6t" width="640" height="452"></p>

**Comment by bateller** [comment, marked as answer]
<p>Not sure if anything has changed with the Dark Theme, but I haven’t been experiencing this issue the last day or so.</p>
<p>I haven’t changed anything on my end.</p>

**Comment by hellaAnonymous** [comment]
<p>Had that same problem, it recovered itself after 4 days, but for me I chose dark mode, still same thing but the opposite. P.S. It hasn’t happen in 9 days.</p>

**Comment by cehfisher** [comment]
Hi @bateller @queenofcorgis - this issue seems to have been resolved [per the comment from the original poster](https://github.com/orgs/community/discussions/23995#discussioncomment-3242448). Could you please confirm and close this issue? Thanks!
